jelares/portfolio/src/main/webapp/javascript/script.js

```diff
@@ -0,0 +1,153 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Changes the text of an element to a randomly chosen string
```

Comment:
I think it could be helpful to note that the elementId is not itself used, but rather elementId1 and elementId2. When documenting a function consider what info the user of this function would need to know, without having to read through the implementation.

Example:
```
@param {string[]} textChoices An array of strings from which the element text is randomly chosen.
@param {string} elementId The text is set on either elementId + '1' or elementId + '2'. <Also describe what determines which element the text is set on, as this is not obvious>.
```

Reply:
sounds good, I made the docstring much more specific